### DISCUSSION TITLE
Error when running dpgen to the end of 0 7

### DISCUSSION THREAD

**Discussion by Baopidragon1** [deepmodeling/dpgen#1435] [Q&A]
When dpgen runs to the end of the fp calculation, it gives an error when labeling and generating new training data.
<img width="86" alt="process2" src="https://github.com/deepmodeling/dpgen/assets/154771059/1688c99b-5981-4824-806c-f9ca77a76bd3">
<img width="687" alt="process" src="https://github.com/deepmodeling/dpgen/assets/154771059/ffe3618c-aaa5-46be-8449-ca3f38b8186f">
<img width="926" alt="bug" src="https://github.com/deepmodeling/dpgen/assets/154771059/3dbebb6c-4462-495f-b6f2-61565148926e">
[param.json](https://github.com/deepmodeling/dpgen/files/13763542/param.json)
[machine.json](https://github.com/deepmodeling/dpgen/files/13763544/machine.json)

**Comment by njzjz** [comment]
Same as #1237
